models/article-models.js

```diff
@@ -30,4 +30,20 @@ exports.selectAllArticles = () => {
         }
         return rows;
     });
+}
+
+
+exports.selectPatchArticleById= ( newVote , article_id) => {
+    return db.query(`
+        UPDATE articles
+        SET votes = $1
+        WHERE article_id = $2
+        RETURNING *;
+    `,[article_id, newVote]).then(({rows}) => {
+        if (rows.length === 0) {
+            return Promise.reject({ status: 404, msg: 'Not Found' });
+        }
+        return rows;
+    });
```

Comment:
Very good. However, SET votes = $1 will set the votes to newVote, it won't increment the current vote. For this you'll need SET votes = votes +$1

Please check that your test reflects this too :)